source/hands_on/trex.rst

```diff
@@ -0,0 +1,82 @@
+TREX
+^^^^^^^
+
+Overview
+---------
+TREX (Trajectory-ranked Reward Extrapolation) was first proposed in `Extrapolating Beyond Suboptimal Demonstrations via Inverse Reinforcement Learning from Observations <https://arxiv.org/abs/1904.06387>`_, which uses ranked demonstrations to inference reward function. Different from previous methods, TREX seeks a reward function that explains the ranking over demonstrations, rather than justifying the demonstrations. Hence, this algorithm can learn good policies from highly suboptimal demostrations.
+
+Quick Facts
+-------------
+1. Demonstrations used in TREX require **ranking** information.
+
+2. The reward function optimization can be viewed as a simple binary classification problem.
+
+3. Usually, multiple data augmentation methods should be applied to prevent reward network from over-fitting.
+
+Key Equations or Key Graphs
+---------------------------
+The loss for reward function:
+
+.. math::
+
+   \mathcal{L}(\theta)=\mathbb{E}_{\tau_i, \tau_j \sim \Pi}[\xi(\mathcal{P}(\hat J_\theta(\tau_i) < \hat J_\theta(\tau_j)),\tau_i \prec \tau_j)]

```

Comment:
It might be better to explain more about how these ranking traj are generated, especially in out DI-engine implementation (i.e. the experiments you carried out below)